### DISCUSSION TITLE
Open AI 密钥怎么设置？

### DISCUSSION THREAD

**Discussion by Ken4551** [ZuodaoTech/everyone-can-use-english#1195] [Q&A]
我可以设置我自己的吗？还是说必须要购买呢？ 还有我怎么导出文字的pdf 文档呢？

**Comment by an-lee** [comment]
可以用自己的 key，`基本设置` 里 默认 AI 引擎选 OpenAI，在 `高级设置` 里设置自己的密钥。

语音文本处右上角有个菜单栏，里面有个 `打印` 选项。
